packages/types/src/types.ts

```diff
@@ -105,7 +105,7 @@ export interface IHash extends U8a { }
 export type CodecTo = 'toHex' | 'toJSON' | 'toString' | 'toU8a';
 
 export interface Constructor<T = Codec> {
-  Fallback?: Constructor<T>;
+  Fallback?: Constructor<Codec>;
```

Comment:
needed to change here. This is more correct, as e.g. EventRecord.Fallback == EventRecord0To76, which is not T